world/achievement_mutation_manager.cpp

```
@@ -0,0 +1,430 @@
+			return TargetResolution::RetryableFailure;
+		}
+		for (auto row : result) {
+			const auto character_id = ParseUInt32(row[0]);
+			if (character_id) {
+				character_ids.push_back(character_id);
+			}
+		}
+		break;
+	}
+	case TargetType::DynamicZone: {
+		const auto dynamic_zone = DynamicZone::FindDynamicZoneByID(
```

Clang (14.0.6)

[warning] 'const auto dynamic_zone' can be declared as 'auto *const dynamic_zone'

(readability-qualified-auto)